docker/docker-compose.yaml

```diff
@@ -0,0 +1,170 @@
+version: '3'
+services:
+    zookeeper:
+        image: wurstmeister/zookeeper
+        networks:
+            - backend
+        expose:
+            - "2181"
+        restart: unless-stopped
+    kafka:
+        image: wurstmeister/kafka:0.10.2.0
```

Comment:
We can either use it or build our own in-tree, which I would be fine with but it'll take a bit more time. We could base it on the alpine variants of openjdk.